services/generic/meter.md

```diff
@@ -1,91 +1,170 @@
-
-#### Interface storage
-
-Name        | Value example | Description
-------------|---------------|-------------
-`sub_value` | "kWh"         | With usage of sub_value, Vinculum will know that it has to store separate meter reading for each unit.
-
-#### Service props
-
-Name                | Value example                                  | Description
---------------------|------------------------------------------------|-------------
-`sup_units`         | ["W", "kWh", "A", "V"]                         | List of supported import/consumption units.
-`sup_export_units`  | ["W", "kWh", "A", "V"]                         | List of supported export/production units.
-`sup_extended_vals` | [See extended report](#extended-report-object) |
-
-#### Extended report object
-
-Name            | Unit    | Description
-----------------|---------|--------------
-`e_import`      | kWh     | Energy Import
-`e_export`      | kWh     | Energy Export
-`last_e_export` | kWh     | Energy Export that day
-`last_e_import` | kWh     | Energy Import that day
-`p_import`      | W       | Power Import
-`p_import_react` | VAR    | Reactive Power Import
-`p_import_apparent` | VA  | Apparent Power Import
-`p_import_avg`  | W       | Power Import average
-`p_import_min`  | W       | Power Import minimum that day
-`p_import_max`  | W       | Power Import max that day
-`p_export`      | W       | Power Export
-`p_export_react` | VAR    | Reactive Power Export
-`p_export_min`  | W       | Power Export minimum that day
-`p_export_max`  | W       | Power Export max that day
-`p_factor`      |         | Power Factor
-`freq`          | Hz      | Frequency
-`freq_min`      | Hz      | Frequency Min
-`freq_max`      | Hz      | Frequency Max
-`u1`            | V       | Voltage phase 1
-`u2`            | V       | Voltage phase 2
-`u3`            | V       | Voltage phase 3
-`i1`            | A       | Current phase 1
-`i2`            | A       | Current phase 2
-`i3`            | A       | Current phase 3
-`dc_p`          | W       | DC Power
-`dc_p_min`      | W       | DC Power
-`dc_p_max`      | W       | DC Power
-`dc_u`          | V       | DC Voltage
-`dc_u_min`      | V       | DC Min Voltage
-`dc_u_max`      | V       | DC Max Voltage
-`dc_i`          | A       | DC Current
-`dc_i_min`      | A       | DC Min Current
-`dc_i_max`      | A       | DC Max Current
\ No newline at end of file
+# Meter Services
+
+Meters report readings of imported/consumed or exported/produced values of their respective type.
+An electricity meter service can represent a stand-alone AMS meter, like a HAN sensor or a main electricity meter, or just a metering functionality of another device.
+
+## Service names
+
+| Service name    | Example units                                 | Description        |
+|-----------------|-----------------------------------------------|--------------------|
+| `meter_elec`    | W, kWh, V, A, kVAh, Hz, power_factor, pulse_c | Electricity meter. |
```

Comment:
I would not use dots (.) at the end, it make anything more clear (applies everywhere)

Reply:
Description is a sentence. Sentences end with period. A loose infinitive or two nouns are usually more confusing than a proper sentence. Once again this documentaion is for both core developer, other developers, support and advanced users.